src/chrome/content/rules/QQ.xml

```diff
@@ -341,6 +342,9 @@
 		<target host="web.weixin.qq.com" />
 		<target host="webpush.weixin.qq.com" />
 	<target host="weixin110.qq.com" />
+	<target host="wj.qq.com" />
+	<target host="img1.write.qq.com" />
```

Comment:
Please also update the [comments](https://github.com/EFForg/https-everywhere/pull/8999/files#diff-45be56cd5bb4a0a24678a54cfcaea0ceL158)